claim_reward_distribute_voters.md

```diff
@@ -152,6 +152,7 @@ query {
       amount
     }
     count
+    }
```

Comment:
Good catch. Please also help indent the subfields exist, rewardDistribution, and count inside the bookkeeping function.